### PR TITLE
story_cycle updates

### PR DESCRIPTION
**Pull request by sfeldkamp**
add enum[duration] fields
add chance field
make fallback optional

### CODE REVIEW

File: config/common/story_cycles.cwt [hunk 1]
```diff
@@ -36,6 +45,7 @@ story_cycle = {
                 effect = single_alias_right[effect_clause]
             }
         }
+        ## cardinality = 0..1
         fallback = single_alias_right[effect_clause]
```
Comment (sfeldkamp) [inline config/common/story_cycles.cwt:49]: None of the game stor_cycle effect_group's make use of fallback.  I think it should be optional.

File: config/common/story_cycles.cwt [hunk 2]
```diff
@@ -10,6 +10,15 @@ story_cycle = {
     on_owner_death = single_alias_right[effect_clause]
     ## cardinality = 0..inf
     effect_group = {
+        ## cardinality = 0..1
+        enum[duration] = int
```
Comment (sfeldkamp) [inline config/common/story_cycles.cwt:14]: I'm not exactly clear yet on the difference between `int` and `int_value_field`.  `enum[duration]` actually uses both throughout this project.  Let me know if this should be changed.

**Comment by sfeldkamp** [comment]
@tboby,

Not sure if you are notified when a PR is added.  Let me know if there is anything else I need to do for this to be considered for merging.

File: config/common/story_cycles.cwt (hunk 2 above)
Comment (IhateTrains) [reply config/common/story_cycles.cwt:14]: Not completely sure (I'm just a collaborator), but I think the difference is that `int_value_field` allows references to e.g. script values (`add_tyranny = some_scripted_value_name`), while `int` accepts only a plain number (`add_tyranny = 420`).
https://github.com/cwtools/cwtools/wiki/.cwt-config-file-guidance#simple-rules

If you haven't noticed any story cycle that uses a named reference for duration value, I think it's OK to keep `int`.

**Review by IhateTrains — APPROVED** [review]
Thanks for the PR!

**Pull request merged**